### PR TITLE
[de] Add missing git submodules init to german README.

### PR DESCRIPTION
**Pull request by hikhvar**
I followed the German Readme to run the local development. Hugo could not compile the website. After reading the English Readme, I saw the missing git submodules init.

In this PR I add that missing submodules init.

### CODE REVIEW

**Comment by coderanger** [comment]
@hikhvar Hi there, we are currently in a high-spam situation due to Hacktoberfest. My German is minimal but this looks like a legit change so I'm going to leave it open, but I would recommend you edit in a description in the first comment just to make it clear this isn't spam.

**Comment by hikhvar** [comment]
Thx for the review @coderanger. Actually this is a drive by change, since I ran into issues setting up the docs locally while following the German Readme.

File: README-de.md [hunk 1]
```diff
@@ -37,6 +37,12 @@ Um die Kubernetes-Website lokal laufen zu lassen, empfiehlt es sich, ein speziel
 
 > Wenn Sie die Website lieber lokal ohne Docker ausführen möchten, finden Sie weitere Informationen unter [Website lokal mit Hugo ausführen](#Die-Site-lokal-mit-Hugo-ausführen).
 
+Das benötigte Docsy Hugo Theme muss als git submodule installiert werden:
```
Comment (mkorbi) [inline README-de.md:40]: Das benötigte [Docsy Hugo Theme](https://github.com/google/docsy#readme) muss als git submodule installiert werden:

File: README-de.md [hunk 2]
```diff
@@ -37,6 +37,12 @@ Um die Kubernetes-Website lokal laufen zu lassen, empfiehlt es sich, ein speziel
...
+
+```
```
Comment (mkorbi) [inline README-de.md:42]:
```
#Installiere die Abhängigkeiten
yarn
```

Comment (mkorbi) [reply README-de.md:42]: At least yarn is in the EN also mentioned

File: README-de.md [hunk 3]
```diff
@@ -37,6 +37,12 @@ Um die Kubernetes-Website lokal laufen zu lassen, empfiehlt es sich, ein speziel
...
+git submodule update --init --recursive --depth 1
```
Comment (mkorbi) [inline README-de.md:43]:
```
#Füge das Docsy submodule hinzu
git submodule update --init --recursive --depth 1
```

Comment (mkorbi) [reply README-de.md:43]: the whole thing with the comment before as one code block :)

File: README-de.md [hunk 4]
```diff
@@ -55,6 +61,12 @@ make container-serve
 
 Hugo-Installationsanweisungen finden Sie in der [offiziellen Hugo-Dokumentation](https://gohugo.io/getting-started/installing/). Stellen Sie sicher, dass Sie die Hugo-Version installieren, die in der Umgebungsvariablen `HUGO_VERSION` in der Datei [`netlify.toml`](netlify.toml#L9) angegeben ist.
 
+Das benötigte Docsy Hugo Theme muss als git submodule installiert werden:
+
+```
+git submodule update --init --recursive --depth 1
+```
+
```
Comment (mkorbi) [inline README-de.md:69]: Isn't the make command enough?

**Review by mkorbi — COMMENTED** [review]
some remarks, lets discuss :)

File: README-de.md (hunk 1 above)
Comment (hikhvar) [reply README-de.md]: Done

File: README-de.md (hunk 3 above)
Comment (hikhvar) [reply README-de.md:44]: Done

File: README-de.md (hunk 2 above)
Comment (kbhawkey) [reply README-de.md:42]: Hi @hikhvar ,
The `en` version of the file has a pending update.
For further information see PR #22995 .

**Comment by irvifa** [comment]
/retitle [de] Add missing git submodules init to german README.

**Comment by mkorbi** [comment]
may I can ask you @hikhvar to double check with the EN PR https://github.com/kubernetes/website/pull/22995/files if it is covered

**Comment by hikhvar** [comment]
@mkorbi I checked the english readme and adjusted this one to the english. PTAL.

**Comment by kbhawkey** [comment]
Thanks @hikhvar .
/approve

**Pull request merged**